f8a_worker/workers/bookkeeper.py

```diff
@@ -71,11 +71,14 @@ def execute(self, arguments):
         self._strict_assert(arguments.get('data'))
 
         aggregated = ''
-        if arguments['data'].get('api_name') == 'stack_analyses' and \
+        if arguments.get('api_name') == 'stack_analyses' and \
            'email' in arguments['data'].get('user_profile', {}):
             aggregated = self.parent_task_result('GraphAggregatorTask')
             self.store_user_node(arguments, aggregated)
 
         postgres = StoragePool.get_connected_storage('BayesianPostgres')
-        postgres.store_api_requests(arguments.get('external_request_id'),
-                                    arguments.get('data'), aggregated)
+        if arguments.get('api_name') == 'component_analyses_post':
+            postgres.store_api_requests_post(arguments)
+        else:
```

Comment:
I am not sure if this flow is still being used, if not can we delete it.

Reply:
if we get Ca get calls it will be used.Otherwise it wont.

Reply:
resolved